tests/components/hvv_departures/test_config_flow.py

```diff
@@ -0,0 +1,258 @@
+"""Test the HVV Departures config flow."""
+from asynctest import patch
+from pygti.exceptions import CannotConnect, InvalidAuth
+
+from homeassistant.components.hvv_departures import config_flow
+from homeassistant.components.hvv_departures.const import CONF_STATION
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+
+
+async def test_user_flow(hass):
+    """Test that config flow works."""
+    flow = config_flow.ConfigFlow()
+    flow.hass = hass
+
+    with patch(
+        "homeassistant.components.hvv_departures.config_flow.GTIHub.authenticate",
+        return_value=True,
+    ), patch(
+        "pygti.gti.GTI.checkName",
+        return_value={
+            "returnCode": "OK",
```

Comment:
Please store the api return values as json fixtures and load those into variables before use.